docs/specification/draft/basic/security_best_practices.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Security Best Practices
+---
+
+## 1. Introduction
+
+### 1.1 Purpose and Scope
+
+This document provides security considerations for the Model Context Protocol (MCP), complementing the MCP Authorization specification. This document identifies security risks, attack vectors, and best practices specific to MCP implementations.
+
+The primary audience for this document includes developers implementing MCP authorization flows, MCP server operators, and security professionals evaluating MCP-based systems. This document should be read alongside the MCP Authorization specification and [OAuth 2.0 security best practices](https://datatracker.ietf.org/doc/html/rfc9700).
+
+## 2. Attacks and Mitigations
+
+This section gives a detailed description of attacks on MCP implementations, along with potential countermeasures.
+
+### 2.1 Confused Deputy Problem
+
+An attacker can exploit configurations where an MCP server operates as a proxy in front of another resource server, leading to the confused deputy problem.
+
+#### 2.1.1 Terminology
+
+**MCP Proxy Server**
+: An MCP server that acts as an intermediary between MCP clients and a protected 
+  third-party API. The MCP proxy server provides MCP functionality while delegating
+API operations to a third-party API server.  The MCP proxy server acts as a single OAuth client to the third-party API server.
+
+**Third-Party Authorization Server**
+: The authorization server that protects access to the third-party API. This server may not
```

Comment:
Also can be trimmed down a bit:

```
Authorization server that protects the third-party API.
It may lack dynamic client registration support, requiring MCP
proxy to use a static client ID for all requests.
```